demos/common/models/src/segmentation_model.cpp

```diff
@@ -19,6 +19,11 @@
 
 using namespace InferenceEngine;
 
+SegmentationModel::SegmentationModel(const std::string& modelFileName, bool useAutoResize) :
+    ModelBase(modelFileName),
+    useAutoResize(useAutoResize) {
```

Comment:
Resize method needs to be specified: https://github.com/openvinotoolkit/open_model_zoo/blob/97c02ad141ae6fd76320372a7b1074aa31629f79/demos/common/models/src/detection_model_ssd.cpp#L96